packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -558,7 +559,10 @@ export class BrowserCriClient {
         browserClient: this.browserClient,
       })
 
-      await this.protocolManager?.connectToBrowser(this.currentlyAttachedTarget)
+      if (!this.currentlyAttachedProtocolTarget) {
+        this.currentlyAttachedProtocolTarget = await this.currentlyAttachedTarget.clone()
```

Comment:
Suggest adding a comment why a cloned target is needed.

Reply:
Added: [`e619cb2` (#31239)](https://github.com/cypress-io/cypress/pull/31239/commits/e619cb23c7d692f22ef0e29370b9b5d8555e6a72)